ansible/roles_ocp_workloads/ocp4_workload_lpe_datagrid/templates/datagrid-secret.yaml.j2

```diff
@@ -0,0 +1,16 @@
+kind: Secret
+metadata:
+  name: {{ ocp_username }}-credentials
+  namespace: "{{ ocp4_workload_lpe_datagrid_namespace }}"
+labels:
+  type: middleware
+type: Opaque
+stringData:
+  identities.yaml: |-
+    credentials:
+    - username: developer
+      password: developer
+    - username: user01
+      password: user01
+    - username: mysecondusername
+      password: changeme-two
```

Comment:
Passwords I would define in the defaults/main.yaml using a variable. And then set them in the AgnosticV config as you're deploying.

Reply:
removed and added it in defaults main file